app/templates/eventView.html

```diff
@@ -10,6 +10,13 @@
     <div class="text-center mb-3">
     {% set page_title = eventData.name %}
         <h1 id="title">{{page_title}}</h1>
+        {% if eventData.name == 'All Volunteer Training' and program == None%}
+          <h3> An All Volunteer Training Event</h3>
+        {% elif program != None%}
+          <h3> A {{program.programName}} Event</h3>
```

Comment:
Take the A out because in the instance of Adopt-A-Grandparent it is "A Adopt-A-Grandparent Event" which is grammatically incorrect. I think "Adopt-A-Grandparent Event" looks good too.

Also, I think it may be redundant to say All Volunteer Training right under All Volunteer Training so you could remove that.

Reply:
Removed A before the program name.

Got rid of All Volunteer Training under All Volunteer Training Event